### PR TITLE
Quasi-static integration in pressure

### PR DESCRIPTION
**Pull request by TheBB**
This makes most of the tests (which are quasistatic) too simple since the fluid drains completely and immediately. I switched all of them to dynamic mode, and the results for all of them look sensible.

Have not looked at downstream effects yet.

### CODE REVIEW

File: Test/Plaxis1DVerif-qstatic.reg [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+	Porosity, n = 0.5
+Number of elements    81
+Number of nodes       265
+Number of dofs        409
+Number of D-dofs      288
+Number of P-dofs      121
+Number of unknowns    352
+  step=1  time=8475.82
+  Primary solution summary: L2-norm            : 0.430464
+                            Max Y-displacement : 1 node 144
```
Comment (TheBB) [inline Test/Plaxis1DVerif-qstatic.reg]: Is there a reason why the pressure isn't printed if it's zero? Seems like a sensible thing to check here.

File: PoroElastic/PoroElasticity.C [hunk 2]
```diff
@@ -358,9 +358,10 @@ bool PoroElasticity::evalInt (LocalIntegral& elmInt,
                                 scl*alpha*fe.detJxW))
     return false;
 
-  if (!this->evalCompressibilityMatrix(elMat.A[pp_S], fe.basis(2),
-                                       scl*scl*Minv*fe.detJxW))
-    return false;
+  if (m_mode == SIM::DYNAMIC)
+    if (!this->evalCompressibilityMatrix(elMat.A[pp_S], fe.basis(2),
+                                        scl*scl*Minv*fe.detJxW))
```
Comment (kmokstad) [inline PoroElastic/PoroElasticity.C]: one more space in l363 to align with elMat i l362.

File: PoroElastic/PoroMats.C [hunk 3]
```diff
@@ -52,7 +51,7 @@ const Matrix& PoroElasticity::Mats::getNewtonMatrix () const
             <<"\nElement compressibility matrix, S_pp"<< A[pp_S]
             <<"\nElement permeability matrix, P_pp"<< A[pp_P];
 #endif
-  const_cast<Matrix&>(A[pp_S]).add(A[pp_P], h);
+  const_cast<Matrix&>(A[pp_S]).add(A[pp_P], 1.0);
```
Comment (kmokstad) [inline PoroElastic/PoroMats.C]: Can skip the `, 1.0` since that is the default scaling

File: Test/Plaxis1DVerif-qstatic.reg (hunk 1 above)
Comment (kmokstad) [reply Test/Plaxis1DVerif-qstatic.reg]: No particular reason, guess it was that someone thought it was unnecessary to check quantities that should be identically zero (inherited from Elasticity). But I agree that for the pressure it is wise to check this. So replace SIMPoroElasticity.h:l88-90 with the following:
```
     IFEM::cout <<"\n                            Max pressure       : "
                << utl::trunc(dMax[nsd]) <<" node "<< iMax[nsd] <<"\n";'
```
We still need the utl::trunc deviise to make sure values less than a certain tolerance is printed as 0.

**Comment by TheBB** [comment]
This PR is updated and needs reapproval. I made the statisticity (?) of the flow switchable, it now accepts `-fullstatic` (default) or `-halfstatic` as command-line options, so I was able to leave most of the tests alone.

Note: This does not address the problem I talked about on Skype this morning.

**Comment by TheBB** [comment]
Was merged as part or #37

**Pull request closed without merging**